src/client/components/Heading/Heading.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './heading.style.css';
+
+import PropTypes from 'prop-types';
+
+const Header = ({ children }) => <header className="header">{children}</header>;
```

Comment:
Why not name the file/folder `Header` to match? It's hard to find this component since it doesn't match!

Reply:
As said above folder vs file name matching is helpful. 
![image](https://user-images.githubusercontent.com/61273043/100395874-fd440400-3042-11eb-96f3-0a71309bbbad.png)